packages/contracts/contracts/StabilityPool.sol

```diff
@@ -69,10 +69,6 @@ import "./Dependencies/console.sol";
  * So, to track P accurately, we use a scale factor: if a liquidation would cause P to decrease to <1e-9 (and be rounded to 0 by Solidity),
  * we first multiply P by 1e9, and increment a currentScale factor by 1.
  *
- * The added benefit of using 1e9 for the scale factor (rather than 1e18) is that it ensures negligible precision loss close to the 
```

Comment:
Remind me to discuss 1e9 vs 1e18 with you

Reply:
not sure why this got deleted ill add it back into the next PR